tapir/api/urls.py

```diff
@@ -0,0 +1,30 @@
+from django.urls import path, include
+from rest_framework import routers
+
+# Serializers define the API representation.
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView,
+    TokenRefreshView,
+    TokenVerifyView,
+)
+
+from tapir.api.views import UpcomingShiftAttendanceView, ShareOwnerView, TapirUserView
+
+# Routers provide an easy way of automatically determining the URL conf.
+router = routers.DefaultRouter()
+
+# Wire up our API using automatic URL routing.
+# Additionally, we include login URLs for the browsable API.
+urlpatterns = [
+    path("", include(router.urls)),
+    path(
+        "upcoming_shift_attendance/",
+        UpcomingShiftAttendanceView.as_view(),
+        name="get_upcoming_shift_attendance",
+    ),
+    path("share_owner/", ShareOwnerView.as_view(), name="get_share_owner"),
+    path("tapir_user/", TapirUserView.as_view(), name="get_tapir_user"),
```

Comment:
(nit) I feel we should just call this `user/`. It's just that our custom class happens to be called `TapirUser`, but I feel this is an implementation detail that should not surface in our external API design.

Reply:
agreed!